test/structures/globals.lua

```diff
@@ -0,0 +1,33 @@
+config.mode = 'title'
```

Comment:
why is this restricted to the title screen? It looks pretty generic

Reply:
I copied it from `find.lua`, which is similarly generic.

Reply:
Thinking about it a little more, `find.lua` is not safe to run when a world is loaded because it permanently modifies world state - that's probably why it was configured that way. `globals.lua` does clean up after itself properly.